source/slang/slang-ir-early-intrinsic-simplification.cpp

```diff
@@ -0,0 +1,169 @@
+// slang-ir-ssa-simplification.cpp
+#include "slang-ir-early-intrinsic-simplification.h"
```

Comment:
Let's use a more specific file name, since the file is currently doing a specific thing.